src/style.css

```diff
@@ -1 +1,33 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 22px;
+}
+
+.stars {
+  display: inline-block;
+  align-items: center;
+  margin-bottom: 10px;
+ 
+}
```

Comment:
The '.stars' class has 'align-items: center;' which is not effective because 'display: inline-block;' does not create a flex container. Consider using 'display: flex;' if you want to align items within this container.